RAW/bitacora.tps

```diff
@@ -0,0 +1,293 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data version="1.0">
+    <struct type="Settings">
+        <key>fileFormatVersion</key>
+        <int>4</int>
+        <key>texturePackerVersion</key>
+        <string>4.4.0</string>
+        <key>fileName</key>
+        <string>D:/Developer/Games/Nya/RAW/bitacora.tps</string>
```

Comment:
This is the way the files are saved but it actually uses relative paths, https://www.codeandweb.com/texturepacker/documentation
See Data file section